.github/workflows/merge-develop-into-list-command.yml

```diff
@@ -0,0 +1,13 @@
+name: Merge the develop branch into the list-command branch
+
+on:
+  schedule:
+    - cron: '* 1 * * *'
```

Comment:
IIRC GitHub Actions only run automatically on default branch (i.e. develop branch). Will this schedule work?